app/src/main/java/com/example/android/codelabs/agera/UiUtils.java

```diff
@@ -21,18 +21,18 @@
 import android.view.animation.Animation;
 import android.view.animation.TranslateAnimation;
 
-public class UiUtils {
+public class UIUtils {
```

Comment:
I checked the document, this point you're right:

![image](https://cloud.githubusercontent.com/assets/5214214/16766082/66540e1a-486a-11e6-9374-5a2a903dfbd3.png)

https://source.android.com/source/code-style.html